lua/autorun/client/cl_cfc_restart_display.lua

```diff
@@ -0,0 +1,71 @@
+local finalTime
+local keyFrame = 0
+local isOpen = false
+local flashState = false
+local CFCRestart = {}
+surface.CreateFont( "CFC_CALIBRI_20PX_DEFAULT", {
+    font = "Calibri",
+    extended = false,
+    size = 40,
+    weight = 500
+})
+
+local function timeLeft()
+    return finalTime - SysTime()
+end
+
+local function restartUI()
+    timer.Create( "CFC_RESTART_FLASHTIMER", 1, 0, function()
+        flashState = not flashState
+    end )
+
+    isOpen = true
+    hook.Remove( "HUDPaint", "CFCDrawRestartAlert" )
+    local delay = net.ReadInt( 16 )
```

Comment:
Not a huge fan of this being here, I personally prefer things like this to be arguments in the function, and passed in by the net receiver.
This would mean changing the receiver to something like this:
`net.Receive( "CFC_RESTART_START", function()
    local delay = net.ReadInt( 16 )
    restartUI( delay )
end )`

The purpose of this is to not restrict `restartUI` to being a net receiver, with this change it could be used elsewhere. While this is unlikely in this csae, it's generally a good practice.

Entirely up for debate though (which I do encourage)